titus-server-master/src/test/java/com/netflix/titus/master/integration/v3/job/JobValidatorNegativeTest.java

```diff
@@ -32,7 +38,12 @@
 public class JobValidatorNegativeTest extends BaseIntegrationTest {
     private static final Logger logger = LoggerFactory.getLogger(JobValidatorNegativeTest.class);
 
-    private static final TitusStackResource titusStackResource = getTitusStackResource(new FailJobValidator());
+    private static final TitusStackResource titusStackResource =
+            getTitusStackResource(
+                    new ParallelValidator(
+                            Duration.ofSeconds(10),  // This is an arbitrary large timeout; the FailJobValidator fails instantaneously, so timeout never occurs.
+                            Collections.emptyList(),
```

Comment:
Minor, might be easier to read as two static list objects `hardValidators = Collections.emptyList()` and `softValidators = Arrays.asList()`